project_task_type/project_demo.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <!-- SET new mandatory field for existing demo data -->
+        <function model="project.task" name="write">
+             <function eval="[[]]" model="project.task" name="search"/>
```

Comment:
@elicoidal  yes,It mandatory the  already exist data's type column to dev。 I don't know what do you mean?